src/main/kotlin/no/nav/familie/pdf/pdf/PdfOppretterController.kt

```diff
@@ -12,33 +11,20 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 @RequestMapping("api")
 class PdfOppretterController {
+    private val pdfOppretterService = PdfOppretterService()
+
     @PostMapping("/lag-pdf")
     fun lagPdf(
         @RequestBody søknad: Map<String, Any>,
-    ): ByteArray {
-        val generertPdf = PdfOppretterService().lagPdf(søknad)
-        return generertPdf
-    }
+    ): ByteArray = pdfOppretterService.lagPdf(søknad)
 
     @CrossOrigin(origins = ["http://localhost:5173"])
     @PostMapping("generate-pdf")
     fun lagPdfMedValidering(
         @RequestBody søknad: Map<String, Any>,
-    ): PdfMedStandarder {
-        val pdf =
-            PdfOppretterService()
-                .lagPdf(søknad)
-
-        return lagPdfMedStandarder(pdf)
-    }
+    ): PdfMedStandarder = pdfOppretterService.lagPdfMedStandarder(søknad)
 
     @CrossOrigin(origins = ["http://localhost:5173"])
     @GetMapping("lag-pdf-med-standarder")
-    fun hentPdfFraResourceMedStandarder(): PdfMedStandarder {
-        val søknad = lesJSON()
-        val pdf =
-            PdfOppretterService()
-                .lagPdf(søknad)
-        return lagPdfMedStandarder(pdf)
-    }
+    fun hentPdfFraResourceMedStandarder(): PdfMedStandarder = pdfOppretterService.lagTestPdfMedStandarder()
```

Comment:
Det ser ut som at dette endepunktet kun er ment for testing, basert på metodenavnet. Jeg ville lagt slike enedepunkter i en egen controller. For eksempel: `TestPdfController`